packages/stream_video_flutter/lib/src/livestream/livestream_content.dart

```diff
@@ -6,16 +6,34 @@ import '../l10n/localization_extension.dart';
 
 typedef LivestreamHostsUnavailableBuilder = Widget Function(
   BuildContext context,
-  Call call,
+  LivestreamHostsUnavailableProperties properties,
 );
 
+class LivestreamHostsUnavailableProperties {
+  LivestreamHostsUnavailableProperties(
+    this.call,
+  );
+
+  final Call call;
+}
+
 typedef LivestreamNotConnectedBuilder = Widget Function(
   BuildContext context,
-  Call call,
-  bool isMigrating,
-  bool isReconnecting,
+  LivestreamNotConnectedProperties properties,
 );
 
+class LivestreamNotConnectedProperties {
+  LivestreamNotConnectedProperties(
+    this.call,
+    this.isMigrating,
+    this.isReconnecting,
+  );
+
+  final Call call;
+  final bool isMigrating;
+  final bool isReconnecting;
```

Comment:
I would always make basic types like `bool`, `int` and `string` (required) named parameters, otherwise it's too easy to switch them around by accident.